tests/php/Core/EnvironmentTest/test.env

```diff
@@ -0,0 +1,13 @@
+# Test file
```

Comment:
There are no tests for nested variables, eg:

```
TEST="value"
TEST_AGAIN="$TEST"
```

Reply:
I didn't write any code to support it either.

Reply:
Are nested variables a thing? :o

Reply:
please don't let this be a thing... at least not here :P

Reply:
My thoughts exactly.

Reply:
This is a feature of the dotenv lib so if we're replacing it like-for-like it needs to be here.

Reply:
I've restored this. ;)